client/electron/windows/OutlineService/OutlineService/OutlineService.cs

```diff
@@ -39,21 +39,21 @@
  *
  * Requests
  *
- * configureRouting: Modifies the system's routing table to route all traffic through the TAP device
- * except that destined for proxyIp. Disables IPv6 traffic.
+ * configureRouting: Modifies the system's routing table to route all traffic through the TAP devicep.
+ * Disables IPv6 traffic. proxyIp is used to find the best gateway adapter.
```

Comment:
Do we still need to disable IPv6? Why?

Reply:
I guess not. But the service code seems to be very dependent on IPv4 addresses. To be safe, I would not change that for this PR.